src/main/java/com/airbnb/billow/Handler.java

```diff
@@ -243,17 +395,72 @@ private void handleComplexEC2(HttpServletResponse response,
         return tables;
     }
 
-    Collection<DynamoTable> sortTablesWithExpression(final Collection<DynamoTable> tables,
-                                                        final String expression)
+    Collection<SQSQueue> listQueuesFromQueryExpression(final String expression, final AWSDatabase db)
+        throws OgnlException {
+        final Collection<SQSQueue> allQueues = db.getSqsQueues().values();
+        if (expression == null)
+            return allQueues;
+
+        final Object compiled = Ognl.parseExpression(expression);
+        final List<SQSQueue> queues = new ArrayList<>();
+
+        for (SQSQueue queue : allQueues) {
+            final Object value = Ognl.getValue(compiled, queues);
+            if (value instanceof Boolean && (Boolean) value)
+                queues.add(queue);
+        }
+
+        return queues;
+    }
+
+    Collection<ElasticacheReservedCacheNodesOffering> listReservedCacheNodeOfferingFromQueryExpression(final String
+                                                                                                    expression, final AWSDatabase db)
```

Comment:
I am using the airbnb java style file but the variable name is too long, I will remove this cuz we don't need "cache node offerings" any more